### PR TITLE
Promote from BLAIS5-3212-refactor-b to main

### PR DESCRIPTION
**Pull request by social-surveys-blaise-concourse**
Auto generated by concourse:

Promote from BLAIS5-3212-refactor-b to main

### CODE REVIEW

File: models/appointment_resource_planning_model.py [hunk 1]
```diff
@@ -38,60 +39,60 @@ def get_appointments_for_date(cls, config, date, survey_tla, questionnaires):
...
+            FROM cati.DaybatchCaseInfo AS dbci
+                LEFT JOIN DialHistory dh
+                    ON dh.InstrumentId = dbci.InstrumentId
+                    AND dh.PrimaryKeyValue = dbci.PrimaryKeyValue
+                    AND dh.DialResult = "Appointment"
+                INNER JOIN UniqueDialHistoryIdTable uid
+                    ON dh.id = uid.id
+            WHERE dbci.AppointmentType != "0"
+            AND dbci.AppointmentStartDate LIKE "{date}%"
             GROUP BY
```
Comment (Riceyo) [inline models/appointment_resource_planning_model.py:87]: Group by is now redundant as no longer using an aggregate.

File: models/appointment_resource_planning_model.py [hunk 2]
```diff
@@ -38,60 +39,60 @@ def get_appointments_for_date(cls, config, date, survey_tla, questionnaires):
 
         query = f"""
             with UniqueDialHistoryIdTable as
```
Comment (Riceyo) [inline models/appointment_resource_planning_model.py:41]: "with" and "as" are keywords so could be capitalised... #nitpicking :p

File: models/appointment_resource_planning_model.py [hunk 3]
```diff
@@ -38,60 +39,60 @@ def get_appointments_for_date(cls, config, date, survey_tla, questionnaires):
...
-                    configuration.Configuration cf
-                ON dh.InstrumentId = cf.InstrumentId
-                WHERE {questionnaire_filter}
-                GROUP BY
-                    dh.PrimaryKeyValue,
-                    dh.InstrumentId)
-                
-            select
+            (SELECT 
+                max(Id) as id,
```
Comment (Riceyo) [inline models/appointment_resource_planning_model.py:43]: "as" is a keyword so could be capitalised... #nitpicking :p

**Review by Riceyo — DISMISSED** [review]
good stuff, just some very minor tweaks...

File: models/appointment_resource_planning_model.py (hunk 1 above)
Comment (elthorne) [reply models/appointment_resource_planning_model.py]: You're absolutely right, I've made the amendments and successfully tested.

**Review by elthorne — APPROVED** [review]

**Pull request merged**